python/psautohint/__main__.py

```diff
@@ -385,6 +386,35 @@ def __call__(self, parser, namespace, values, option_string=None):
         parser.exit(message=formatter.format_help())
 
 
+class _DeprecatedAction(argparse.Action):
+    """
+    Custom action to raise a DeprecationWarning when hit. Used for the
+    autohint compatibility args/flags.
+    """
+    def __init__(self,
```

Comment:
add blank line above `def __init__` (and also at line no. 369)

Reply:
done